docs/creating-manifest/visual-settings.md

```diff
@@ -673,67 +673,140 @@ values:
 @@!
 
 ### dockertags
-Field for displaying Docker tags within the list element.
+Field for displaying **Docker tags** within the *[list](#list)* element.
 
-![text](/img/dockertag.jpg)
+![text](/img/dockertags.png)
+
+The *tags* of specific *nodeType* can be displayed like in the wizard with no acquiring from the server:   
```

Comment:
it's not clear what 'wizard' means

Reply:
the sentence changed like this:
The *tags* of specific *nodeType* can be displayed like in the wizard above with no acquiring from the server. - That means that image that is displayed just before this sentence is a mentioned wizard.
![image](https://user-images.githubusercontent.com/39057925/70060887-fc24f600-15eb-11ea-8c3d-bacc6929b483.png)

Reply:
image replaced with another type: tomcat tags